packages/markers/src/browser/problem/problem-contribution.ts

```diff
@@ -0,0 +1,65 @@
+/*
+* Copyright (C) 2017 TypeFox and others.
+*
+* Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+*/
+
+import { injectable, inject } from 'inversify';
+import { ProblemWidget } from './problem-widget';
+import {
+    MenuModelRegistry, Command, CommandContribution,
+    MenuContribution, KeybindingContribution, KeybindingRegistry,
+    KeyCode, Key, Modifier, CommandRegistry, MAIN_MENU_BAR
+} from '@theia/core/lib/common';
+import { CommonCommands, FrontendApplication } from '@theia/core/lib/browser';
+
+export const MARKER_CONTEXT_MENU = 'marker-context-menu';
+
+export namespace ProblemCommands {
+    export const OPEN: Command = {
+        id: 'markers:open',
+        label: 'Markers'
+    };
+}
+
+@injectable()
+export class ProblemContribution implements CommandContribution, MenuContribution, KeybindingContribution {
+
+    constructor(
+        @inject(ProblemWidget) protected readonly markerWidget: ProblemWidget,
+        @inject(FrontendApplication) protected readonly app: FrontendApplication) {
+    }
+
+    registerKeyBindings(keybindings: KeybindingRegistry): void {
+        keybindings.registerKeyBinding({
+            commandId: ProblemCommands.OPEN.id,
+            keyCode: KeyCode.createKeyCode({
+                first: Key.KEY_M, modifiers: [Modifier.M2, Modifier.M1]
```

Comment:
Humm will this be seen as Markers or Problems by the user I wonder ?... for the usage of key 'M'

Reply:
It's the same binding as in vs code. Also `P` is heavily used by open resource and open command.

Reply:
OK